pkg/accounting/db.go

```diff
@@ -42,14 +50,14 @@ type Rollup struct {
 type DB interface {
 	// LastTimestamp records the latest last tallied time.
 	LastTimestamp(ctx context.Context, timestampType string) (time.Time, error)
-	// SaveBWRaw records raw sums of agreement values to the database and updates the LastTimestamp.
-	SaveBWRaw(ctx context.Context, tallyEnd time.Time, created time.Time, bwTotals map[storj.NodeID][]int64) error
 	// SaveAtRestRaw records raw tallies of at-rest-data.
 	SaveAtRestRaw(ctx context.Context, latestTally time.Time, created time.Time, nodeData map[storj.NodeID]float64) error
 	// GetRaw retrieves all raw tallies
 	GetRaw(ctx context.Context) ([]*Raw, error)
-	// GetRawSince r retrieves all raw tallies sinces
+	// GetRawSince retrieves all raw tallies since latestRollup
 	GetRawSince(ctx context.Context, latestRollup time.Time) ([]*Raw, error)
+	// GetBWSince retrieves all bandwidth_rollup entires since latestRollup
+	GetBWSince(ctx context.Context, latestRollup time.Time) ([]*BW, error)
```

Comment:
i think this name should be more descriptive as well.  Maybe `GetStoragenodeBandwidthRollups` or `GetStoragenodeBandwidthSince`? I know the names are getting long, but its just confusing because there are `StoragenodeBandwidthRollup` and also `BucketBandwidthRollup` so I would like to be specific which we are talking about.

Reply:
good idea to both rename suggestions